packages/features/bookings/lib/handleNewBooking/test/recurring-event.test.ts

```diff
@@ -40,219 +41,776 @@ describe("handleNewBooking", () => {
+              eventTypeId: mockBookingData.eventTypeId,
+              status: BookingStatus.ACCEPTED,
+              recurringEventId: mockBookingData.recurringEventId,
+              references: [
+                {
+                  type: "daily_video",
+                  uid: "MOCK_ID",
+                  meetingId: "MOCK_ID",
+                  meetingPassword: "MOCK_PASS",
+                  meetingUrl: "http://mock-dailyvideo.example.com/meeting-1",
+                },
+                {
+                  type: "google_calendar",
+                  uid: "MOCKED_GOOGLE_CALENDAR_EVENT_ID",
+                  meetingId: "MOCKED_GOOGLE_CALENDAR_EVENT_ID",
+                  meetingPassword: "MOCK_PASSWORD",
+                  meetingUrl: "https://UNUSED_URL",
+                },
+              ],
+            });
+
+            expectBookingCreatedWebhookToHaveBeenFired({
+              booker,
+              organizer,
+              location: "integrations:daily",
+              subscriberUrl: "http://my-webhook.example.com",
+              //FIXME: File a bug - All recurring bookings seem to have the same URL. They should have same CalVideo URL which could mean that future recurring meetings would have already expired by the time they are needed.
+              videoCallUrl: `${WEBAPP_URL}/video/${createdBookings[0].uid}`,
+            });
+          }
+
+          // expectWorkflowToBeTriggered();
+
+          expectSuccessfulBookingCreationEmails({
+            booking: {
+              // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+              uid: createdBookings[0].uid!,
+              urlOrigin: CAL_URL,
+            },
+            booker,
+            organizer,
+            emails,
+            iCalUID: "MOCKED_GOOGLE_CALENDAR_ICS_ID",
+            recurrence: {
+              ...recurrence,
+              count: recurringCountInRequest,
+            },
+          });
+
+          expect(emails.get().length).toBe(2);
+
+          expectSuccessfulCalendarEventCreationInCalendar(calendarMock, [
+            {
+              calendarId: "event-type-1@google-calendar.com",
+              videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
+            },
+            {
+              calendarId: "event-type-1@google-calendar.com",
+              videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
+            },
+            {
+              calendarId: "event-type-1@google-calendar.com",
+              videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
+            },
+            {
+              calendarId: "event-type-1@google-calendar.com",
+              videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
+            },
+          ]);
+        },
+        timeout
+      );
+    });
+    describe("Round robin event type:", () => {
+      test("should when when a fixed host is not available on the second slot", async () => {
```

Comment:
this would have failed before, as we didn't check the availability of the fixed host for the next slots